### PR TITLE
Fixes and changelog

### PR DESCRIPTION
**Pull request by dantleech**
Small corrections and changelog entry for #165

### CODE REVIEW

File: Tests/Unit/Adapter/PhpcrOdmAdapterTest.php [hunk 1]
```diff
@@ -203,17 +203,9 @@ public function testFindRouteForUri()
...
         $expectedRoute = $this->route->reveal();
 
-        $this->dm->find(null, $this->baseRoutePath . $uri)->willReturn($expectedRoute);
+        $this->dm->find('Symfony\Cmf\Component\RoutingAuto\Model\AutoRouteInterface', $this->baseRoutePath . $uri)->willReturn($expectedRoute);
 
         $res = $this->adapter->findRouteForUri($uri, $this->uriContext->reveal());
         $this->assertSame($expectedRoute, $res);
     }
-
-    public function testFindRouteForUriShouldReturnNullWhenNodeAtGivenPathIsNotAnAutoRoute()
```
Comment (dantleech) [inline Tests/Unit/Adapter/PhpcrOdmAdapterTest.php]: This is now the responsibility of the document manager.

File: Tests/Unit/Adapter/PhpcrOdmAdapterTest.php [hunk 2]
```diff
@@ -184,7 +184,7 @@ public function provideCompareRouteContent()
     public function testCompareRouteContent($isMatch)
     {
         $this->route->getContent()->willReturn($this->contentDocument);
-        $content = $isMatch ? $this->contentDocument : $this->contentDocument2;
+        $isMatch ? $this->contentDocument : $this->contentDocument2;
```
Comment (dantleech) [inline Tests/Unit/Adapter/PhpcrOdmAdapterTest.php]: `$content` was not used.

**Comment by dantleech** [comment]
@mmenozzi With regards to the exception test I had assumed that the exception was thrown in a public method, so what you have is fine.

**Comment by mmenozzi** [comment]
@dantleech ok!

**Pull request merged**